release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -45,6 +45,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-8-rhcos-butane"]
+==== Butane config transpiler simplifies creation of machine configs
```

Comment:
Here and the next paragraph - should machine config be capitalized? Similiar to how it is here: https://docs.openshift.com/container-platform/4.7/rest_api/machine_apis/machineconfig-machineconfiguration-openshift-io-v1.html

Reply:
@sagidlow - No, that API doc is auto-generated and not under docs guideline governance. In this case, `machine config` is what our guidelines consider a [general reference](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#general-references) that should be two words and in lowercase. Thanks for checking!